src/parking-locations/helper-functions/geoHashHelpers.ts

```diff
@@ -0,0 +1,41 @@
+import ngeohash from 'ngeohash';
+import { AugmentedParkingLocationData } from '../types';
+
+const geohashPrecision = 5;
```

Comment:
NIT: A comment as to what this entails and/or how it was established would be nice :) 

Q: Do I understand correctly that this sets the size of the geohashed regions that are included as the user's and neighboring regions? If so, how big is a region with precision = 5?

Reply:
understood!